site/content/includes/quickstart-http-query-steps.md

```diff
@@ -0,0 +1,128 @@
+To query metrics, use the _{{% apiendpoint %}}query_range_ endpoint with the following data in the request body, all fields are required:
```

Comment:
Mention the instant query api?

Reply:
@benraskin92 This partial is used quite widely, that could be useful. Can you remind me of the endpoint and parameters?

Reply:
https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries -- basically same as query_range but doesnt take a start, end or step

Reply:
OK, great, I think I might save this for a different PR though, as it will affect a lot of different guides